app/modules/encounters/resources.py

```diff
@@ -174,19 +95,90 @@ def get(self, encounter):
     )
     @api.login_required(oauth_scopes=['encounters:write'])
     @api.parameters(parameters.PatchEncounterDetailsParameters())
-    @api.response(schemas.DetailedEncounterSchema())
     @api.response(code=HTTPStatus.CONFLICT)
     def patch(self, args, encounter):
         """
         Patch Encounter details by ID.
         """
+
+        edm_count = 0
+        for arg in args:
+            if (
+                'path' in arg
+                and arg['path']
+                in parameters.PatchEncounterDetailsParameters.PATH_CHOICES_EDM
+            ):
+                edm_count += 1
+        if edm_count > 0 and edm_count != len(args):
+            log.error(f'Mixed edm/houston patch called with args {args}')
+            abort(
+                success=False,
+                passed_message='Cannot mix EDM patch paths and houston patch paths',
+                message='Error',
+                code=400,
+            )
+
+        rdata = {}
+        if edm_count > 0:
+            log.debug(f'wanting to do edm patch on args={args}')
+            headers = {
```

Comment:
This block is now duplicated in both the sightings and the encounters code.
Could current_app.edm.request_passthrough be tweaked to optionally take the request.headers and generate the headers once.
This does seem really edm specific so that seems a better place for it.

Reply:
header stuff has been moved/consolidated